snippets/get-transcription-result.mdx

```diff
@@ -20,18 +20,24 @@ You can get your transcription results in **3 different ways**:
 
   Callback are HTTP calls that you can use to get notified when your transcripts are ready.
 
-  Instead of polling and keeping your server busy and maintaining work, you can use the `callback_url` feature.
-  By providing a `callback_url` on your transcription request body payload, we will send the results right there once 
+  Instead of polling and keeping your server busy and maintaining work, you can use the `callback` feature.
+  By providing a `callback_config` field along with a `callback` field set to `true` on your transcription request body payload, we will send the results right there once 
   your transcription is done.
```

Comment:
`By providing...` is a bit redundant with the sentence coming right after the example.  Maybe we can remove it ?
```suggestion  
  Instead of polling and keeping your server busy and maintaining work, you can use the `callback` feature to receive the result to a specified endpoint:
```